crates/node/build.rs

```diff
@@ -1,4 +1,9 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
+    vergen::EmitBuilder::builder()
+        .build_timestamp()
+        .all_git()
+        .emit()?;
```

Comment:
Capture information from build so that we can identify which versions of the node are running in the network.